.github/workflows/ubuntu-ci.yml

```diff
@@ -39,21 +49,19 @@ concurrency:
 jobs:
   ubuntu-ci:
 
-    strategy:
-      fail-fast: false
-      matrix:
-        os-image:
-          - 'ubuntu-22.04'
-
     if: ${{ (
               !contains(github.event.pull_request.labels.*.name, 'skip-ci') &&
               !contains(github.event.pull_request.labels.*.name, 'conflicts')
             ) }}
     uses: ./.github/workflows/reusable-ubuntu-ci.yml
     with:
-      os-image: ${{ matrix.os-image }}
+      # It would be desirable to have a matrix of ubuntu OS for this job, but due to the issue opened in this ticket:
+      # https://github.com/orgs/community/discussions/128118 , it has been set as a single OS job.
+      os-image: 'ubuntu-22.04'
       label: ${{ inputs.label || 'ubuntu-ci' }}
       colcon-args: ${{ inputs.colcon-args }}
-      cmake-args: '-DSECURITY=ON ${{ inputs.cmake-args }}'
+      cmake-args: ${{ inputs.cmake-args }}
       ctest-args: ${{ inputs.ctest-args || '-LE xfail' }}
       fastdds-branch: ${{ inputs.fastdds_branch || github.ref || 'master' }}
+      security: ${{ inputs.security || true }}
+      run-tests: ${{ inputs.run-tests || !contains(github.event.pull_request.labels.*.name, 'no-test') }}
```

Comment:
I think that since the input has a default this is always going to be true regardless of the label. Please double check